docs/docs/api/Dispatcher.md

```diff
@@ -194,7 +194,7 @@ Returns: `Boolean` - `false` if dispatcher is busy and further dispatch calls wo
 * **method** `string`
 * **reset** `boolean` (optional) - Default: `false` - If `false`, the request will attempt to create a long-living connection by sending the `connection: keep-alive` header,otherwise will attempt to close it immediately after response by sending `connection: close` within the request and closing the socket afterwards.
 * **body** `string | Buffer | Uint8Array | stream.Readable | Iterable | AsyncIterable | null` (optional) - Default: `null`
-* **headers** `UndiciHeaders | string[]` (optional) - Default: `null`.
+* **headers** `UndiciHeaders` (optional) - Default: `null`.
```

Comment:
I'm not sure we should have undici types here? @mcollina

Reply:
I can rename to `OutgoingHttpHeaders` also. 
I saw [this](https://github.com/nodejs/undici/blob/a427e4b948c4fdae8d86a013565c3929111601b2/docs/docs/api/Dispatcher.md?plain=1#L1308) in the docs which matched the type

Reply:
any thoughts on this @mcollina ?

Reply:
I think we should define `UndiciHeaders` somewhere if we do.